luaui/Widgets/gui_defenserange_gl4.lua

```diff
@@ -254,7 +254,20 @@ local function initUnitList()
 
 		-- LEGION
```

Comment:
T1 LLT (`leglht`) and T1 arty (`legcluster`) are missing from this list.

Reply:
what weapon types do leglht and legcluster have?
0 = default / land/ground based weapon
1 = AA
2 = anti-nuke
3 = long-range artillery

Reply:
`leglht` should be 0, it's just the LLT equivalent. Pretty sure `legcluster` is also 0 since it's a short range arty as an equivalent to the Agitator and Gauntlet.